src/de/grobox/liberario/TransportNetworks.java

```diff
@@ -406,6 +406,12 @@ public TransportNetworks(Context context) {
 				         .setStatus(TransportNetwork.Status.BETA)
 		);
 
+		list.add(new TransportNetwork(context, NetworkId.OREGON)
+                                .setName(getString(R.string.np_name_oregon))
```

Comment:
Style Nitpick: this line has spaces, but should have tabs. You can see it is misaligned with the following lines.